cloud

```diff
@@ -1260,6 +1265,7 @@ function cloud_functions_env() {
                           database \
                           clean_test_databases \
                           empty_test_databases \
+                          schema \
```

Comment:
I don't think we need to pass the schema version to the environment. We could discover it from the databases themselves, if needed.

Reply:
Okay